docs/integrations/spark/spark.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 3
+sidebar_position: 1
```

Comment:
😉

Reply:
This jumped deeper into category tree ;-)